### PR TITLE
Chore: removed update_assets method from API

### PR DESCRIPTION
**Pull request by mallok-s**
Removed update_assets method as the web editor has been removed from datapane.com

## Prerequsites

- [x] Has been tested locally
- [ ] If a bugfix, have included a breaking test if possible

## Proposed Changes

- Removing the update_assets & update methods from the API as the web editor is gone now

### CODE REVIEW

**Review by mands — APPROVED** [review]
LGTM - I think you may be able to clean up some of the code in `core.Report.upload` as part of this now we've removed `update_assets` - as we split things out to handle both scenarios previously

**Comment by mallok-s** [comment]
There's a problem with removing the update function. The schelduling tests fails. Are we keeping the user's ability to scheldule reports? Or is this also being removed?

File: projects/python-client/src/datapane/client/api/dp_object.py [hunk 1]
```diff
@@ -187,13 +187,6 @@ def delete(self):
         self.res.delete()
         log.debug(f"Deleted object {self.url}")
 
-    def update(self, **kwargs):
```
Comment (mands) [inline projects/python-client/src/datapane/client/api/dp_object.py:190]: Sorry - I missed this earlier, but I don't think this method should be removed, it's a generic method to perform a `patch` http request to the backend

Comment (mallok-s) [reply projects/python-client/src/datapane/client/api/dp_object.py]: No worries, I also missed this! I've added it back in and the tests are happy now

**Review by mands — APPROVED** [review]
LGTM

**Pull request merged**